lib/kaffe/producer.ex

```diff
@@ -83,6 +97,11 @@ defmodule Kaffe.Producer do
   ## internal
   ## -------------------------------------------------------------------------
 
+  defp produce_list(topic, partition, message_list) when is_list(message_list) do
+    Logger.debug "event#produce_list topic=#{topic} partition=#{partition}"
+    @kafka.produce_sync(client_name(), topic, partition, "ignored", message_list)
```

Comment:
@rwdaigle I wondered the same thing when I was writing the code,  I will try to do that...